mcs/class/corlib/System.Reflection/AssemblyName.cs

```diff
@@ -456,7 +456,10 @@ void _AssemblyName.GetTypeInfoCount (out uint pcTInfo)
 				return (cultureinfo == null)? null : cultureinfo.Name;
 			}
 			set {
-				throw new NotImplementedException ();
+				if (value == null)
+					cultureinfo = null;
+				else
+					cultureinfo = new CultureInfo (value);
```

Comment:
Is this right? elsewhere we do `CultureInfo.CreateCulture (name)` which looks like it has some non-trivial logic in it.

Reply:
I mostly wrote this based on the code we had previously, so I wouldn't be shocked if it's wrong.

Reply:
I think the PR is fine as-is, let's worry about other edge cases once we get a bug about it ;)